common/entrypoint_mautic_cron.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ ! -f /opt/mautic/cron/mautic ]; then
+
+	cat <<EOF > /opt/mautic/cron/mautic
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+BASH_ENV=/tmp/cron.env
+
+* * * * * php /var/www/html/bin/console mautic:segments:update 2>&1 | tee /tmp/stdout
```

Comment:
Is there a reason why only this is added to cron?

Documentation mentions at least about few other commands that are required
https://docs.mautic.org/en/5.x/configuration/cron_jobs.html

Reply:
@lumnn this was addressed, the default crons are present, and can be changed via a cron file